.github/workflows/re-build-boards.yml

```diff
@@ -38,6 +35,7 @@ jobs:
       with:
```

Comment:
Rename this job to `board`. That should make the task name change from `Build CI / boards (atmel-samd) / build (8086_commander) (pull_request) ` to `Build CI / ports (atmel-samd) / board (8086_commander) (pull_request)`